contracts/SlashIndicator.sol

```diff
@@ -96,9 +98,13 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     }
     indicator.height = block.number;
     if (indicator.count % felonyThreshold == 0) {
-      indicator.count = 0;
-      IBSCValidatorSet(VALIDATOR_CONTRACT_ADDR).felony(validator);
-      ICrossChain(CROSS_CHAIN_CONTRACT_ADDR).sendSynPackage(SLASH_CHANNELID, encodeSlashPackage(validator), 0);
+      try ICrossChain(CROSS_CHAIN_CONTRACT_ADDR).sendSynPackage(SLASH_CHANNELID, encodeSlashPackage(validator), 0) {
+          indicator.count = 0;
+          IBSCValidatorSet(VALIDATOR_CONTRACT_ADDR).felony(validator);
+      } catch (bytes memory reason) {
+          indicator.count--;
```

Comment:
Because it is punished, so no need to count down.

Reply:
fixed